### PR TITLE
Add proof of concept support to vulnerability

### PR DESCRIPTION
**Pull request by stevespringett**
Added proof of concept to vulnerability containing steps to reproduce, environment, and attachments of supporting material.

### CODE REVIEW

**Comment by stevespringett** [comment]
Closes #119

File: schema/bom-1.5.proto [hunk 1]
```diff
@@ -604,6 +604,17 @@ message Vulnerability {
   repeated Property properties = 18;
   // The date and time (timestamp) when the vulnerability record was rejected (if applicable).
   optional google.protobuf.Timestamp rejected = 19;
+  // Evidence used to reproduce the vulnerability.
+  optional ProofOfConcept proofOfConcept = 20;
+}
+
+message ProofOfConcept {
+  // Precise steps to reproduce the vulnerability.
+  optional string reproductionSteps = 1;
```
Comment (prabhu) [inline schema/bom-1.5.proto:613]: Thoughts on accepting workarounds and mitigation recommendations?

Comment (stevespringett) [reply schema/bom-1.5.proto:613]: recommendations are already part of the spec. Would workarounds be a separate field independent of recommendations?

Comment (prabhu) [reply schema/bom-1.5.proto:613]: Yes, will be nice to have workarounds separately from mitigations.

Comment (stevespringett) [reply schema/bom-1.5.proto:615]: Workarounds have been added to the spec and test cases updated.

**Review by jkowalleck — APPROVED** [review]

**Pull request merged**